ui/text_displayer.js

```diff
@@ -168,6 +168,133 @@ shaka.ui.TextDisplayer = class {
     }
   }
 
+  /**
+   * Returns a safe collection of html elements for display as a single cue.
+   * @param {string} cueBody
+   * @return {Array<HTMLElement>}
+   * @private
+   */
+  buildCueElements_(cueBody) {
+    const tokenize = (str) => {
+      if (!str) {
```

Comment:
This also doesn't seem to be necessary.  If you don't check for an empty string, you'd still get an empty array of tokens, right?

Reply:
Per above, and resolved.